libyul/optimiser/OptimizerUtilities.cpp

```diff
@@ -38,3 +43,9 @@ void yul::removeEmptyBlocks(Block& _block)
 	};
 	boost::range::remove_erase_if(_block.statements, isEmptyBlock);
 }
+
+// TODO: Should this be part of the Dialect instead?
```

Comment:
Fine with me either way - but maybe yes.
I'd also be fine with inlining this into the two use cases - and with keeping it here :-).

Reply:
I think it fits better here. The dialect should not be concerned about keywords.

Reply:
Removed.